test/ng/intervalSpec.js

```diff
@@ -281,6 +281,36 @@ describe('$interval', function() {
   });
 
 
+  describe('cancelAll', function() {
+    it('should cancel all pending intervals', inject(function ($interval, $window) {
+      var counter = 0;
+      var promise = $interval(function () {
```

Comment:
Consider adding more than one intervals as you are testing cancelAll function, so it will make sense to have atleast 3 intervals with different periodic time and cancelAll should cancel all of them